### PR TITLE
Try to fix CI sctp test.

### PR DESCRIPTION
**Pull request by slontis**
For some reason when I tried this before it worked. So retrying this again to see if it still works.
Not sure why it decided to fail once merged yet.

##### Checklist

- [ ] documentation is added or updated
- [x] tests are added or updated

### CODE REVIEW

**Comment by slontis** [comment]
This is a test -(mainly for my own sanity)

**Comment by slontis** [comment]
@paulidale - this verifies what we saw before.. i.e. it works here... Maybe that means the infrastructure is different for the daily build?

File: .github/workflows/run-checker-daily-sctp.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+jobs:
+  run-checker:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Dependencies for sctp option
+      run:  |
+        sudo apt-get update
+        sudo apt-get -yq install lksctp-tools libsctp-dev
+        checksctp
```
Comment (paulidale) [inline .github/workflows/run-checker-daily-sctp.yml:28]: This should be enough and get rid of the OK environment variable:
```yaml
- name: Check SCTP is available
  run: checksctp
```

File: .github/workflows/run-checker-daily-sctp.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      run: ./configdata.pm --dump
+      if: $OK == 0
+
+    - name: make
+      run: make -s -j4
+      if: $OK == 0
+
+    - name: make test
+      run: make test HARNESS_JOBS=${HARNESS_JOBS:-4}
+      if: $OK == 0
```
Comment (paulidale) [inline .github/workflows/run-checker-daily-sctp.yml:49]: New line at end of file.

**Review by paulidale — COMMENTED** [review]
I can't see a difference.  Maybe add this to `ci.yml` instead?

File: .github/workflows/run-checker-daily-sctp.yml (hunk 1 above)
Comment (slontis) [reply .github/workflows/run-checker-daily-sctp.yml:28]: Except for when it fails again on merge and I have to spend another day looking at it :)

Comment (paulidale) [reply .github/workflows/run-checker-daily-sctp.yml:28]: Yeah, that.  This step will fail if checksctp does and the rest won't run.  It avoid the conditionals.

Comment (slontis) [reply .github/workflows/run-checker-daily-sctp.yml:26]: We might have to try this out by merging..

File: .github/workflows/run-checker-daily-sctp.yml [hunk 3]
```diff
@@ -26,24 +24,24 @@ jobs:
...
-        echo ${{ OK }}
+
     - name: Enable SCTP auth
+      id: sctp_auth
+      continue-on-error: true
       run: sudo sysctl -w net.sctp.auth_enable=1
-      if: $OK == 0
 
     - name: config
+      if: steps.sctp_auth.outcome == 'success' && steps.sctp_auth.conclusion == 'success'
```
Comment (paulidale) [inline .github/workflows/run-checker-daily-sctp.yml:34]: I'm not convince these are useful still.
A failure terminates the job.

Comment (slontis) [reply .github/workflows/run-checker-daily-sctp.yml:34]: there is a continue-on-error..

**Comment by slontis** [comment]
@paulidale - just entered something bad to test it out.. That skips the tests and marks it as green

**Comment by slontis** [comment]
What I am trying to do here is skip doing the tests if sctp cant be setup, and report a green result..
I would like to try merging this so we can see what happens.. It will still be better than failing as it does currently in the daily build.
If we see that is is still always skipping then we can take it back out again.

**Review by paulidale — APPROVED** [review]

**Comment by slontis** [comment]
ping

**Review by t8m — APPROVED** [review]
Let's try merging this to see.

**Comment by t8m** [comment]
Merging as this is CI failure fix.

**Comment by t8m** [comment]
Merged to master branch. Thank you.

**Pull request closed without merging**